.golangci.reference.yml

```diff
@@ -1111,6 +1111,11 @@ linters-settings:
     # Default: 1
     tab-width: 1
 
+  loggercheck:
+    logr: true
+    klog: true
+    zap: true
+
```

Comment:
If the linter has a configuration, the exhaustive configuration of the linter must be added (alphabetical case-insensitive order)
    - The values must be different from the default ones.
    - The default values must be defined in a comment.
    - The option must have a short description.